Movies-MVP/MoviesMVPTests/RealmServiceTest.swift

```diff
@@ -0,0 +1,44 @@
+// RealmServiceTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+@testable import MoviesMVP
+
+/// Тестирование реалм-сервиса
+final class RealmServiceTests: XCTestCase {
+    // MARK: - Private Properties
+
+    private var realmService: RealmServiceProtocol?
+
+    // MARK: - Public Methode
+
+    override func setUp() {
+        realmService = RealmService()
+    }
+
+    override func tearDown() {
+        realmService = nil
+    }
+
+    override func setUpWithError() throws {}
+    override func tearDownWithError() throws {}
+
+    func testExample() throws {}
+
+    func testPerformanceExample() throws {
+        measure {}
+    }
+
+    func testMovie() {
+        let mockMovies = [Movie()]
+        realmService?.saveData(mockMovies)
+        let result = realmService?.loadMovieData(Movie.self, typeMovie: Url.populary)
+        XCTAssertNotNil(result)
```

Comment:
```suggestion
        XCTAssertNotNil(result)
```
1 XCTest 1 метод разнеси в разные